edalize/vunit_hooks.py

```diff
@@ -0,0 +1,27 @@
+'''This module exports `VUnitHooks` which can be used to implement advanced VUnit test cases'''
+
+from vunit.ui import Library
+from vunit import VUnit
+from typing import Mapping, Collection
+
+class VUnitHooks(object):
+    '''Derive the VUnitRunner instance from this class and override its member functions if necessary'''

```

Comment:
```suggestion
    '''Derive the :class:`VUnitRunner` instance from this class and override its member functions if necessary.'''
```
This will hyperlink the identifier.

Reply:
Thanks for pointing this out to me. I have adjusted this instance and some other places as well.